dapr-spring/dapr-spring-cloudconfig/src/main/java/io/dapr/spring/boot/cloudconfig/autoconfigure/DaprCloudConfigAutoConfiguration.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2016-2024 Team Fangkehou
```

Comment:
@lony2003 you need to use the standard Dapr header for all the new files. https://github.com/dapr/java-sdk/blob/master/dapr-spring/dapr-spring-boot-autoconfigure/src/main/java/io/dapr/spring/boot/autoconfigure/client/DaprClientAutoConfiguration.java#L1